src/ApplePayGateway.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+namespace Omnipay\Vindicia;
+
+
+class ApplePayGateway extends AbstractVindiciaGateway
+{
+
+    /**
+     * Get the gateway parameters.
+     *
+     * @return array
+     */
+    public function getDefaultParameters()
+    {
+        return array(
+            'username' => '',
+            'password' => '',
+            'testMode' => false,
+        );
+    }
+
+    /**
+     * Get the gateway name.
+     *
+     * @return string
+     */
+    public function getName()
+    {
+        return 'Vindicia ApplePay';
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getUsername()
+    {
+        return $this->getParameter('username');
+    }
+
+    /**
+     * @return AbstractVindiciaGateway
+     */
+    public function setUsername($value)
+    {
+        return $this->setParameter('username', $value);
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getPassword()
+    {
+        return $this->getParameter('password');
+    }
+
+    /**
+     * @return AbstractVindiciaGateway
+     */
+    public function setPassword($value)
+    {
+        return $this->setParameter('password', $value);
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getPemCertPath()
+    {
+        return $this->getParameter('pemCertPath');
+    }
+
+    /**
+     * @param string $value
+     * @return static
+     */
+    public function setPemCertPath($value)
```

Comment:
could you add docs explaining what these certs are and what the path should look like?

Reply:
It's a path to the user's personal path, so I'm not sure how I would show what that would look like... I've used `('./certs/path_to_cert.crt.pem');` in the main docblock. Also, there really isn't much information on what the certs are besides the fact that you get them from apple and have to split them. Not sure if I should add that since it's not apart of Appple Pay docs. I got my information from scouring forums/stackoverflow.

Reply:
^^ that example is helpful, you could add it here too

Reply:
i like the docs you added for these files in the request, could you add them here too?

Reply:
Did you mean like this part of the request doc?

https://github.com/vimeo/omnipay-vindicia/pull/42/files#diff-ba17822789aa8e485b85ea6429963389R16

Reply:
I was thinking https://github.com/vimeo/omnipay-vindicia/pull/42/files#diff-ba17822789aa8e485b85ea6429963389R148